README.md

```diff
@@ -103,7 +104,7 @@ Contributions welcome! Please see [CONTRIBUTING](https://github.com/frqnck/aweso
   - [One Open Sky](https://www.oneopensky.dk/)
   - [Special](https://www.specialmilano.com/)
   - [CucinaBarilla](https://www.cucinabarilla.it)
-  = [212Performance](https://www.212performance.com/)
+  - [212Performance](https://www.212performance.com/)
```

Comment:
When I open the page I'm getting a 403 error.

Reply:
No errors here. 

I believe there are IP Blocks to certain IP Addresses that are flagged for some reason. It could be that you're on one of them.